### ISSUE TITLE
Add warning when adding git lfs-tracked files without git lfs

### ISSUE THREAD

**Issue by mrp089** [SimVascular/svMultiPhysics#26]
We use `git lfs` to store large files (`.vtp`, `.vtu`) which is set in `.gitattributes`. This is tracked automatically, so files added with `git add` get automatically tracked.

Unfortunately, developers must first install `git lfs` before using this feature. If a developer doesn't have `git lfs` installed and adds files with `git add`, it adds the actual (potentially large) files to the repository.

I'll add a tool to give a warning when a pull request gets opened that contains actual files that should be tracked with `git lfs` instead of their pointers. This should avoid adding large files to the repository

**Comment by aabrown100-git** [comment]
@mrp089 Our weekly issue-handling group came across this issue. Are you planning on trying again to resolve this issue? If it's not worth it, perhaps we can just make it extra clear in the documentation that you need to install git lfs.

**Comment by aabrown100-git** [comment]
When did the repo go public? It looks like your last attempt was January 24, 2024, which I thought was after the repo went public.

**Comment by mrp089** [comment]
Ah, I didn't remember #170. You can add more documentation, but in the end, it comes down to the people merging a pull request to check if there are no large files committed to the repository. Fixing that retrospectively is extremely annoying (see #149), and it only gets worse the more people are using svFSIplus.

**Comment by aabrown100-git** [comment]
We tried this again with https://github.com/SimVascular/svFSIplus/pull/289, unsuccessfully.

@dcodoni Can you explain here why you think it is failing? Is it because GITHUB_TOKEN permits access on SimVascular/svFSIplus, but the integration test check is being run using a forked account?

I was thinking, it would probably be enough if the the git lfs check was run on all forked repositories. That way, as soon as the developer makes a commit to their own fork with large files that should be tracked by git lfs, they will receive an error. Is this any easier?

**Comment by dcodoni** [comment]
@aabrown100-git I think the test fails because the GITHUB_TOKEN automatically detect Simvascular as the owner and for this reason we git lfs check does not have permissions to operate on a personal fork.

If we really want this check to be done, each fork should have a workflow file that performs the check on a pull request and it uses a personal token. The downside of this is that each one of us should be responsible for creating his own personal token and give it the same name as in the workflow file. After a merge to the main though, the check will fail.

**Comment by aabrown100-git** [comment]
I asked for help from the lfs-warning developers [here](https://github.com/ppremk/lfs-warning/issues/147#issuecomment-2438913689), but have yet to get a response. Going to give up for now.

**Comment by ktbolt** [comment]
@aabrown100-git  Are we going to get this working or shall I close this ?

**Comment by dcodoni** [comment]
I think we should close it, what do you think @aabrown100-git

**Comment by aabrown100-git** [comment]
Yes I think we can close. We should all just be aware of new contributors potentially committing large files untracked by git lfs.
